### PR TITLE
Fixes #6303: Improve the API documentation and bump the default version ...

### PR DESCRIPTION
**Pull request by Kegeruneku**
...to 5

Ticket: http://www.rudder-project.org/redmine/issues/6303

### CODE REVIEW

File: introduction.md [hunk 1]
```diff
@@ -1,96 +1,100 @@
...
 <span id="api-_-Authentication"/>
 ## Authentication
 
+Authenticating against the API is mandatory for every request, as sensitive information like inventories or configuration rules may get exposed.
 
-To use the API, you need to be authenticated. Rudder may contain capital information about your IT, and we don't want that information to be public, do we?
+It is done using a dedicated API Account, than can be created in the web interface on the 'API Accounts' page located inside the Administration part.
 
-Authentication is done via a dedicated API Account. Those accounts can be created in Rudder, on the 'API Accounts' page located inside the Administration part. API Accounts are not connected to standard user accounts, and are currently global, accessible by admins only.
+API Accounts are not linked to standard user accounts, and currently give full administrative privileges: They must be secured adequately.
```
Comment (fanf) [inline introduction.md]: lower "t" for they

File: introduction.md [hunk 2]
```diff
@@ -1,96 +1,100 @@
...
 
 <p><img title="Rudder API Settings page" alt="Rudder API page" src="img/APISettings.png" class="RudderImage"></p>
 
-Once you have created an API Account, you can now use the API. A token is generated for each account which you will need to authenticate.
+Once you have created an API account, you get a token that you will need to authenticate on every request. This token is the API equivalent of a password, and must
```
Comment (fanf) [inline introduction.md]: that will be needed to authenticate each request

**Comment by fanf** [comment]
All in all, this is a very welcome improvement! @Kegeruneku, could you just address the little remarks ?

**Comment by Kegeruneku** [comment]
Updated according to review

**Comment by fanf** [comment]
Great !

**Pull request merged**